Extension/package.nls.json

```diff
@@ -1,4 +1,10 @@
 {
+    "c_cpp.subheaders.intelliSense.title" : "IntelliSense Features",
```

Comment:
Is "Features" needed?  It seems a bit redundant.

Reply:
I think I requested that to help differentiate between the other IntelliSense Default Config one.